Stack/Core/Security/Certificates/CertificateValidator.cs

```diff
@@ -961,12 +1153,12 @@ protected virtual void InternalValidate(X509Certificate2 certificate)
             }
         }
 
-		
-		        /// <summary>
+        /// <summary>
         /// Throws an exception if validation fails.
         /// </summary>
         /// <param name="certificates">The certificates to be checked.</param>
         /// <exception cref="ServiceResultException">If certificate[0] cannot be accepted</exception>
+        [Obsolete("Class is replaced by Opc.Ua.CertificateValidator.InternalValidate(X509Certificate2Collection certificates, ConfiguredEndpoint endpoint)")]
         protected virtual void InternalValidateWithChainSupportEnabled(X509Certificate2Collection certificates)
```

Comment:
I think some ofthese functions better be removed...

Reply:
Don't we risk breaking backward compatibility since they are protected ?